hedera-node/test-clients/src/main/resource/contract/contracts/Module049OpcodesExist/Module049OpcodesExist.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.24;
+
+contract Module049OpcodesExist {
+    function try_transient_storage() public {
+        assembly {
+            tstore(10, 11111)
+            pop(tload(10))
+        }
+    }
+
+    function try_mcopy() public pure {
+        bytes memory byts = "Hello, World!";
+        assembly {
+           mcopy(0,0,32)
+        }
+    }
+
+    function try_kzg_precompile() public view {
+        bytes memory byts = "Hello, World!";
+
+        (bool success, ) = address(0x0A).staticcall(byts);
+        // If KZG precompile _succeeds_ with this bad data is must not be implemented; i.e., expect fail
```

Comment:
Q: is there a simple check to confirm a success?

Reply:
(Discussed offline, result:) This precompile will have an additional test that checks that with a specific canned input data it actually succeeds.  Test brought over from BESU.